x-pack/plugins/security_solution/public/entity_analytics/components/include_closed_alerts_section.tsx

```diff
@@ -0,0 +1,177 @@
+    const savedStart = localStorage.getItem('dateStart');
+    const savedEnd = localStorage.getItem('dateEnd');
+
+    if (savedIncludeClosedAlerts !== null) {
+      setIncludeClosedAlerts(JSON.parse(savedIncludeClosedAlerts));
+    }
+    if (savedStart) {
+      setFrom(savedStart);
+    }
+    if (savedEnd) {
+      setTo(savedEnd);
+    }
+  }, [setIncludeClosedAlerts]);
+
+  const onRefresh = ({ start: newStart, end: newEnd }: { start: string; end: string }) => {
+    setFrom(newStart);
+    setTo(newEnd);
+    onDateChange({ start: newStart, end: newEnd });
+    setShowBar(true);
+
+    localStorage.setItem('dateStart', newStart);
+    localStorage.setItem('dateEnd', newEnd);
+  };
+
+  const handleToggle = () => {
+    const newValue = !includeClosedAlerts;
+    setIncludeClosedAlerts(newValue);
+    setShowBar(true);
+    localStorage.setItem('includeClosedAlerts', JSON.stringify(newValue));
+  };
+
+  const { mutate } = useConfigureSORiskEngineMutation();
+
+  const handleSave = () => {
+    setIsLoading(true);
+
+    mutate(
+      {
+        includeClosedAlerts,
+        range: { start, end },
+      },
+      {
+        onSuccess: () => {
+          setShowBar(false);
+          addSuccess(i18n.RISK_ENGINE_SAVED_OBJECT_CONFIGURATION_SUCCESS, {
+            toastLifeTimeMs: 5000,
+          });
+          setIsLoading(false);
+        },
+        onError: () => {
+          setIsLoading(false);
+        },
+      }
+    );
+
+    // Persist the toggle state in localStorage after saving
+    localStorage.setItem('includeClosedAlerts', JSON.stringify(includeClosedAlerts));
+  };
+
+  return (
+    <>
+      <EuiFlexGroup alignItems="center">
+        <EuiFlexItem grow={false}>
+          <EuiSwitch
+            label="Include closed alerts for risk scoring"
+            checked={includeClosedAlerts}
+            onChange={handleToggle}
+          />
+        </EuiFlexItem>
+        <div
+          className="vertical-line"
+          style={{ height: '24px', borderLeft: '1px solid #ccc', margin: '0 8px' }}
+        />
+        <EuiFlexItem grow={false}>
+          <EuiSuperDatePicker
+            start={start}
+            end={end}
+            onTimeChange={({ start: newStart, end: newEnd }) =>
+              onRefresh({ start: newStart, end: newEnd })
+            }
+            width={'auto'}
+            compressed={false}
+            showUpdateButton={false}
+          />
+        </EuiFlexItem>
+      </EuiFlexGroup>
+      <EuiText size="s" style={{ marginTop: '10px' }}>
+        <p>
+          {`Enable this option to factor both open and closed alerts into the risk engine
+            calculations. Including closed alerts helps provide a more comprehensive risk assessment
+            based on past incidents, leading to more accurate scoring and insights.`}
+        </p>
+      </EuiText>
+      {showBar && (
+        <EuiBottomBar
+          paddingSize="s"
+          position="fixed"
+          css={css`
```

Comment:
Here, you are using emotion-css instead of style 👏

Reply:
Yes, this change was added recently after the design mocks were updated. It occurred to me that using `emotion` would be a better approach 🫣 😉 🙈 , which resulted in a mix of changes. That said, I can still refine this to enhance readability.